### ISSUE TITLE
Panic during creation of Glossary

### ISSUE THREAD

**Issue by turulix** [Avimitin/deepl-rs#27]
Hey! Sorry, its me again. Just noticed the Libary is able to panic. This seems to be caused by the API responding to a request by anything  that isn't expected. In this case using EN_US as a target language, which would return:

```json
{
    "message": "Unsupported glossary source and target language pair"
}
```

I assume having no glossary entries would also panic, as that would return:
```json
{
    "message": "Invalid glossary entries provided"
}
```

```rust
thread 'tokio-runtime-worker' panicked at C:\Users\Tim\.cargo\registry\src\index.crates.io-6f17d22bba15001f\deepl-0.6.0\src\endpoint\glossary.rs:92:26:
Unmatched response to CreateGlossaryResp, please open issue on https://github.com/Avimitin/deepl.: reqwest::Error { kind: Decode, source: Error("missing field `glossary_id`", line: 1, column: 66) }
stack backtrace:
   0: std::panicking::begin_panic_handler
             at /rustc/fdaaaf9f923281ab98b865259aa40fbf93d72c7a/library\std\src\panicking.rs:645
   1: core::panicking::panic_fmt
             at /rustc/fdaaaf9f923281ab98b865259aa40fbf93d72c7a/library\core\src\panicking.rs:72
   2: core::result::unwrap_failed
             at /rustc/fdaaaf9f923281ab98b865259aa40fbf93d72c7a/library\core\src\result.rs:1653
   3: enum2$<core::result::Result<deepl::endpoint::glossary::GlossaryResp,reqwest::error::Error> >::expect<deepl::endpoint::glossary::GlossaryResp,reqwest::error::Error>
             at /rustc/fdaaaf9f923281ab98b865259aa40fbf93d72c7a\library\core\src\result.rs:1034
   4: deepl::endpoint::glossary::impl$1::into_future::async_block$0
             at C:\Users\Tim\.cargo\registry\src\index.crates.io-6f17d22bba15001f\deepl-0.6.0\src\endpoint\glossary.rs:84
   5: core::future::future::impl$1::poll<alloc::boxed::Box<dyn$<core::future::future::Future<assoc$<Output,enum2$<core::result::Result<deepl::endpoint::glossary::GlossaryResp,enum2$<deepl::endpoint::Error> > > > >,core::marker::Send,core::marker::Sync>,alloc::a

```

**Comment by Avimitin** [comment]
Oh, I don't handle edge case and just let them fail. I will fix them up in next release.

**Comment by Avimitin** [comment]
Should be fixed in v0.6.1.
